### PR TITLE
fix image update issue on first save #1034

### PR DESCRIPTION
**Pull request by thrijith**
#1034 fixed image edit changes not reflecting on first save

### CODE REVIEW

File: app/main/controllers/template/rtmedia-actions.php [hunk 1]
```diff
@@ -90,7 +90,7 @@ function rtmedia_image_editor_content( $type = 'photo' ) {
 		$thumb_url = wp_get_attachment_image_src( $media_id, 'thumbnail', true );
 
 		echo '<div id="imgedit-response-' . esc_attr( $media_id ) . '"></div>';
-		echo '<div class="wp_attachment_image" id="media-head-' . esc_attr( $media_id ) . '">' . '<p id="thumbnail-head-' . esc_attr( $id ) . '"><img class="thumbnail" src="' . esc_url( set_url_scheme( $thumb_url[0] ) ) . '" alt="" /></p>' . $modify_button . '</div>'; // @codingStandardsIgnoreLine
+		echo '<div class="wp_attachment_image" id="media-head-' . esc_attr( $media_id ) . '">' . '<p id="thumbnail-head-' . esc_attr( $media_id ) . '"><img class="thumbnail" src="' . esc_url( set_url_scheme( $thumb_url[0] ) ) . '" alt="" /></p>' . $modify_button . '</div>'; // @codingStandardsIgnoreLine
```
Comment (BhargavBhandari90) [inline app/main/controllers/template/rtmedia-actions.php]: @thrijith Use wp_kses_post for `$modify_button` html markup

Ref: https://codex.wordpress.org/Function_Reference/wp_kses_post

Comment (thrijith) [reply app/main/controllers/template/rtmedia-actions.php]: input tag not allowed in `wp_kses_post`

File: templates/media/media-single-edit.php [hunk 2]
```diff
@@ -3,6 +3,8 @@
 	if ( have_rtmedia() ) : rtmedia();
 		if ( rtmedia_edit_allowed() ) {
 			global $rtmedia_media;
+
+			$media_id_nonce = wp_create_nonce( "image_editor-$rtmedia_media->media_id" );
```
Comment (BhargavBhandari90) [inline templates/media/media-single-edit.php]: Add comments for why you did this.

Comment (BhargavBhandari90) [reply templates/media/media-single-edit.php]: Other looks good.

**Pull request merged**